docs/updates/2019-04.md

```diff
@@ -0,0 +1,180 @@
+`qOneAndOnlyOne` will enforce only one selected value. The selection model in catwalk has been updated accordingly.
+
+![screenshot](../images/catwalk-field-select-one-value.gif)
+
+### Visual support for loosely coupled tables
+
+We have added support for loosely couple tables, i.e. tables with a circular reference. Normally this is not intended in
+a data model and catwalk will help out by marking the involved fields with a dashed red line.
+
+![screenshot](../images/catwalk-circular-reference.png)
+
+### Export hypercube data and definition
+
+In one of the incoming feature requests there was an expressed need for exporting hypercube data. It sounded like an
+interesting features so we decided to build support for it but also offer a way to export the hypercube definition to
+Clipboard. Both features can be found in the new icons top right in open hypercubes.
+
+![screenshot](../images/catwalk-export-hypercube.png)
+
+### Save open hypercubes
+
+Something which has irritated us for a while is that all hypercubes created in catwalk gets lost when the window is
+refreshed or closed. To prevent this we now store the hypercube definition in the browser's local storage.
+
+### Keep table headers during scrolling
+
+Previously, when you navigate in large applications, the table headers were lost which caused some confusions on what
+table the fields belonged. With the latest changes we will keep the table names on top as long as there are more fields
+in the table. This is especially good if you have a complex application with a lot of association. In these it isn't
+uncommon to have tables with large gaps between the fields, making it easy to miss the bottom ones.
+
+![screenshot](../images/catwalk-scrolling-headers.gif)
+
+### Various improvements
+
+Apart from the larger fixes we have made a couple of smaller improvements:
+
+* add support for table and field comments
+* handle rendering issues when layout isn't retrieved and while loading the layout
+* Multiple table-names properly displayed in tool tips
+* Fix key field sorting during selection in other fields
+
+## corectl 1.0.0 beta release
+
+The rest of the team worked on reaching a 1.0.0 version of corectl. An API specification was created, commands
+scrutinized and inconsistencies fixed throughout the commands. In the final work we decided upon going from verb-noun to
+noun-verb to better align with other cli tools, improve discoverability and to provide a uniform command structure and
+output. In short this means that all objects (e.g. apps, dimensions, connections, ...) will be found in the command root
+and relevant actions for these objects an level below.
+
+The other commands is divide between the following groups:
+
+* **App Building** commands such as `build` and `reload` used during app creation
+* **App Analysis** commands used to print data model related information as well as `eval` to evaluate expressions
+* **Other** commands specific to corectl
+
+```txt
```

Comment:
Not sure this output adds much value, perhaps link to the help markdown files in the corectl repo instead?

Reply:
I tend to disagree. I think it is easier to understand the **App Analysis** and **Other** description.

Reply:
I meant including those (to drop it all and link to the tool docs instead), feels like it belongs to the corectl documentation and not just mentioned in an update but YMMV.

Reply:
we are saying "We're highlighting a couple of important changes and improvements", which means that we don't have to mention everything we have done..